pkg/restore/restore_test.go

```diff
@@ -2050,10 +2052,7 @@ func TestRestorePersistentVolumes(t *testing.T) {
 				// restored, we'd get an error of "snapshot not found".
 				"provider-1": &volumeSnapshotter{},
 			},
-			podVolumeBackups: []*velerov1api.PodVolumeBackup{
-				builder.ForPodVolumeBackup("velero", "pvb-1").Result(),
-				builder.ForPodVolumeBackup("velero", "pvb-2").Result(),
-			},
+
```

Comment:
is this change intentional? the test case name still says "include podvolumebackups", not exactly sure what the intent is.